app/controllers/posts_controller.rb

```diff
@@ -25,9 +25,19 @@ def edit
   # POST /posts
   def create
     @post = Post.new(post_params)
-
     if @post.save
-      redirect_to @post, notice: 'Post was successfully created.'
+      tag_params.split(",").each do |tag_new|
```

Comment:
Dodatkowo, wazna rzecz - nie zapisuj posta, jesli cos nie poszlo przy zapisie tagow, ale to zostaw sobie na koniec.